### ISSUE TITLE
Fails to load the conversation list

### ISSUE THREAD

**Issue by rickmak** [skygear-demo/react-chat-demo#14]
I follow the to get the demo up and running. Right after login, I got following error:

```
Uncaught (in promise) Error: Objects are not valid as a React child (found: object with keys {_id}). If you meant to render a collection of children, use an array instead or wrap the object using createFragment(object) from the React add-ons. Check the render method of `ConversationPreview`.
```

**Comment by akiroz** [comment]
Cannot reproduce on both local and hosted. @rickmak

**Comment by rickmak** [comment]
Workfine with current master. Fixed at `7371be44`
